main.c

```diff
@@ -224,7 +231,7 @@ main(int argc, char *argv[])
 			    (argc > 1) ? argv[1] : "standard output");
 			break;
 		case 13:
-			warnp("Error reading file: %s", argv[0]);
+			warnp("Error reading file: %s", infilename);
```

Comment:
What if we have trouble reading stdin?  (Entirely possible if someone runs `scrypt enc - < /file/on/disk` and there's an I/O error.)

Reply:
Rebased and pushed a new version that uses the "Error writing file" approach of checking the variable and printing "standard input" if appropriate.